package/src/scripts/helpers/getLocale.ts

```diff
@@ -5,7 +5,9 @@ const capitalizeFirstLetter = (str: string) => `${str.charAt(0).toUpperCase()}${
 
 const getLocaleWeekday = (self: VanillaCalendar, i: number) => {
 	const weekday = new Date(`1978-01-0${i + 1}T00:00:00.000Z`).toLocaleString(self.settings.lang, { weekday: 'short', timeZone: 'UTC' });
+	const longWeekday = new Date(`1978-01-0${i + 1}T00:00:00.000Z`).toLocaleString(self.settings.lang, { weekday: 'long', timeZone: 'UTC' });
 	(self.locale.weekday as string[]).push(capitalizeFirstLetter(weekday));
+	(self.locale.longWeekday as string[]).push(capitalizeFirstLetter(longWeekday));
```

Comment:
another option might be to keep both long/short weekday in a single array of objects, but that might be a breaking change, so perhaps acceptable for 3.0 roadmap?

`const weekday = [{ long: 'Monday', short: 'Mon' }, ...]`